tests/cases/conformance/jsx/tsxSpreadAttributesResolution16.tsx

```diff
@@ -21,7 +21,7 @@ export default function Component(props: ComponentProps) {
 interface AnotherComponentProps {
     property1: string;
     AnotherProperty1: string;
-    property2: boolean;
+    property2: number;
```

Comment:
the expected error for this test case was a "missing property" - but this `property2` has also an incorrect type, so I tweaked this to keep the original error here